src/test/scala/chisel3/tests/AsyncResetRegTest.scala

```diff
@@ -0,0 +1,159 @@
+// See LICENSE for license details.
```

Comment:
How much SiFive code does this incorporate? Does this need to be LICENSE.SiFive?

Reply:
Re-wrote with ExtModule based on SiFive, but refactor names and interface, so hopefully very little direct SiFive stuff now